src/main/java/tds/exam/results/mappers/OpportunityMapper.java

```diff
@@ -49,6 +49,7 @@
             .collect(Collectors.toList());
 
         TDSReport.Opportunity opportunity = new TDSReport.Opportunity();
+        opportunity.setOppId("0");
```

Comment:
Asked @mtrupkin to try out exam id instead to see if that works.  OppId usually refers to the opportunity id.  Not sure that will work or not.